src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.types.ts

```diff
@@ -49,6 +55,30 @@ export type TimeRangeItemAnswer = {
   'data-testid'?: string;
 };
 
+export type SingleSelectPerRowItemAnswer = {
+  activityItem: MultipleSelectionPerRowItem;
+  answer: DecryptedMultiSelectionPerRowAnswer | null;
+  'data-testid'?: string;
+};
+
+export type MultiSelectPerRowItemAnswer = {
+  activityItem: MultipleSelectionPerRowItem;
+  answer: DecryptedMultiSelectionPerRowAnswer | null;
+  'data-testid'?: string;
+};
+
+export type SingleMultiSelectPerRowActivityItem = {
```

Comment:
why did you create a new type instead of usage of combination:
`SingleSelectionPerRowItem & MultipleSelectionPerRowItem` from src\shared\state\Applet\Applet.schema.ts?